### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 403 (Forbidden)

### DISCUSSION THREAD

**Discussion by S0ulf3re** [mastodon/mastodon#18461] [Q&A]
I have a Mastodon droplet and an S3 space on Digitalocean that im trying to use. Right now, I am able to view toots, but I cannot see any images. I am trying to set up the CDN to to work as well, but enabling that from my subdomain causes the whole thing to not load.

This is how my S3 config is currently set, am I missing something?
<img width="1217" alt="Screen Shot 2022-05-19 at 14 38 20" src="https://user-images.githubusercontent.com/50301900/169375776-3d71e72f-bd59-42b5-9d47-94849ed460d3.png">

Here's portion of my error logs (Safari 15.5)
<img width="486" alt="Screen Shot 2022-05-19 at 14 40 12" src="https://user-images.githubusercontent.com/50301900/169376072-1277d66e-690f-49f3-b11b-13733bb6eb77.png">

## My URL's
Main instance: thetooniverse.xyz
Origin: https://the-tooniverse-space.nyc3.digitaloceanspaces.com
CDN: cdn.thetooniverse.xyz

## My config so far

```
S3_ENABLED=true
S3_BUCKET=the-tooniverse-space
S3_HOSTNAME=cdn.thetooniverse.xyz
S3_ENDPOINT=cdn.thetooniverse.xyz
S3_ALIAS_HOST=the-tooniverse-space.nyc3.digitaloceanspaces.com
# CDN_HOST=cdn.thetooniverse.xyz
# Uncommenting this causes the whole site not to load
```

## Some extra notes:

- Changing file listing to "Public" turns the 403 errors into 404 errors

**Comment by rafaelcaricio** [comment]
Did you had any lucky finding the solution to this?

**Reply by MorenoLabs** [reply]
got exactly same issue, have not found a reference yet on how to fix this
